kiwi/storage/subformat/vagrant_base.py

```diff
@@ -0,0 +1,304 @@
+# Copyright (c) 2019 SUSE Linux GmbH.  All rights reserved.
+#
+# This file is part of kiwi.
+#
+# kiwi is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# kiwi is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with kiwi.  If not, see <http://www.gnu.org/licenses/>
+#
+import json
+import os
+import random
+from textwrap import dedent
+from tempfile import mkdtemp
+from string import Template
+
+from kiwi.storage.subformat.base import DiskFormatBase
+from kiwi.command import Command
+
+from kiwi.exceptions import (
+    KiwiFormatSetupError
+)
+
+
+class VagrantConfigTemplate(object):
```

Comment:
ok, nice but could you put it into an extra file below the existing template directory: ```kiwi/storage/subformat/template```